templates/articles/blocks/promoted_pages.html

```diff
@@ -1,47 +1,33 @@
-        {% image promoted_item.teaser_image fill-328x196 as teaser_image_medium %}
-        {% image promoted_item.teaser_image fill-348x208 as teaser_image_large %}
-        {% image promoted_item.teaser_image fill-508x304 as teaser_image_extra_large %}
-
-        {% comment %}
-        Source copied from templates/includes/card-group-secondary-nav.html
-
-        Modified slightly to support data from a streamfield.
-
-        This is not intended to be production-ready code.
-
-        To discuss an approach to allow for reusability going forward.
-        {% endcomment %}
-
-        <li class="col-sm-12 col-md-6 col-lg-4 mb-3">
-            <div class="card-group-secondary-nav">
-                <a href="{{ promoted_item.url }}" class="card-group-secondary-nav__image-link" data-card-type="card-group-secondary-nav" data-card-title="{{ promoted_item.title }}" aria-hidden="true" tabindex="-1">
-                    <div class="card-group-secondary-nav__image">
-                        <picture>
-                            <source media="(max-width: 768px)" srcset="{{ teaser_image_extra_large.url }}">
-                            <source media="(max-width: 991px)" srcset="{{ teaser_image_medium.url }}">
-                            <source media="(max-width: 1199px)" srcset="{{ teaser_image_small.url }}">
-                            <source media="(min-width: 1200px)" srcset="{{ teaser_image_large.url }}">
-                            <img src="{{ teaser_image_small.url }}" alt="{{ image.title }}" class="card-group-secondary-nav__image-fallback">
-                        </picture>
-                    </div>
+            <li class="promoted-pages__card">
+                <a class="promoted-pages__card" href="{{ promoted_item.url }}">
+                    <picture>
+                        {% image promoted_item.teaser_image fill-350x229-c100 format-webp as mobile_webp_image %}
+                        <source media="(max-width: 390px)"
+                                srcset="{{ mobile_webp_image.url }}"
+                                type="image/webp"/>
+                        {% image promoted_item.teaser_image fill-370x229-c100 format-webp as desktop_webp_image %}
+                        <source srcset="{{ desktop_webp_image.url }}" type="image/webp"/>
+                        {% image promoted_item.teaser_image fill-350x229-c100 as mobile_img %}
+                        <source srcset="{{ mobile_img.url }}" media="(max-width: 390px)"/>
+                        {% image promoted_item.teaser_image fill-370x229-c100 as base_img %}
+                        <img src="{{ base_img.url }}"
+                             height="{{ base_img.height }}"
+                             width="{{ base_img.width }}"
+                             class="promoted-pages__image"
+                             {% if promoted_item.teaser_image.alt_text %} alt="{{ promoted_item.teaser_image.alt_text }}"{% endif %}/>
```

Comment:
```suggestion
                             alt="{% if promoted_item.teaser_image.alt_text %} {{ promoted_item.teaser_image.alt_text }}{% endif %}"/>
```

If there is no alt text specified, it's best to leave the alt tag blank rather than removing it - if the alt attribute is missing, the file name will be read instead by default (and if it's called IMG-1824.jpg that's not ideal) - if it's blank, the image will just be ignored instead, which is preferable.